### DISCUSSION TITLE
What's the notify.endpoint setting means in batch expire?

### DISCUSSION THREAD

**Discussion by Davisanity** [minio/minio#20219] [Q&A] [answered]
I would like to ask about the **notify.endpoint** setting in the YAML file generated through **mc batch generate** in the batch framework. What kind of endpoint should be filled in here? Are there any official documents that I can refer to?
![image](https://github.com/user-attachments/assets/ee5db528-4b41-4c59-bb88-afda49935c72)

**Comment by itpey** [comment, marked as answer]
Hi, The notify block in configuration is a tool that ensures you’re always in the loop about the status of your jobs. Whether the job completes successfully or encounters an error, the notify block sends a notification to a specified service, allowing you to take immediate action—whether that’s alerting an administrator, logging the event, or kicking off another process.

To make this work, you need to set the endpoint field in the notify block to a URL that can receive and process these notifications. One effective way to handle these notifications is by **setting up a webhook on your own server**. Alternatively, you can use a dedicated webhook service such as **Zapier**, **IFTTT**, or **Webhook.site**. These services can manage and process incoming notifications.
